README.md

```diff
@@ -33,6 +33,12 @@ C2Rust requires LLVM 6, 7, or 8 with its corresponding clang compiler and librar
 - **Arch Linux:**
 
         pacman -S base-devel llvm clang cmake openssl
+        
+- **NixOS / nix:**
+
+        nix-shell
+        rustup toolchain install nightly
```

Comment:
We depend on a specific nightly (currently nightly-2019-04-12), which rustup should automatically install if you cargo build in the project directory because of the `rust-toolchain` file. You shouldn't need these rustup comands?

Reply:
Ok that makes sense. I am a novice when it comes to rust.